apps/root/src/hooks/useTransactionsHistory.tsx

```diff
@@ -54,22 +51,35 @@ function useTransactionsHistory(): {
         storedWallets.map(({ address }) => address)
       );
 
+      const nonIndexedTransactions = localTransactions.filter(
+        (tx) =>
+          !resolvedEvents.find(
+            (serviceEvent) =>
+              serviceEvent.tx.chainId === tx.chainId && serviceEvent.tx.txHash.toLowerCase() === tx.hash.toLowerCase()
```

Comment:
shouldn't this be parsed with resolvedEvents? since its the final list we are going to show

Reply:
Absolutely!